nautobot_device_lifecycle_mgmt/template_content.py

```diff
@@ -83,7 +83,7 @@ def right_page(self):
         extra_context = {
             "validsoft_table": self.device_software.get_validated_software_table(),
             "obj_soft": self.device_software.software,
-            "obj_soft_valid": self.device_software.validate_software(),
+            "obj_soft_valid": self.device_software.validate_software()[0],
```

Comment:
Thinking this may be better to selecting the object in the template rather than passing it into extra_content?